Update README first line to match new use cases (#31247)
mattermost/mattermost: README.md

# [![Mattermost logo](https://user-images.githubusercontent.com/7205829/137170381-fe86eef0-bccc-4fdd-8e92-b258884ebdd7.png)](https://mattermost.com)

> [Mattermost](https://mattermost.com) is an open core, self-hostable collaboration platform offering persistent chat & ChatOps, workflow, and toolchain automation, integrated voice, screen, file and content sharing, and AI-enhanced information synthesis. This repo is the primary source for core development on the Mattermost platform; it's written in Go and React and runs as a single Linux binary with MySQL or PostgreSQL. A new compiled version is released under an MIT license every month on the 16th.

Reviewer: I'm struggling a bit with this. Lots of it is true, e.g. `open core`, `self-hostable` and the broad feature set. But I have trouble parsing `persistent chat` especially right next to `ChatOps`, and my buzzword alarm goes off with the word `synthesis`. There's also the grammar nit in me that thinks we have to use semicolons to properly separate the sublists, but then that alone feels like a weak start to a paragraph.

Thoughts on this instead?
```suggestion
[Mattermost](https://mattermost.com) is an open core, self-hosted collaboration platform that offers chat, workflow automation, voice calling, screen sharing, and AI integration. This repo is the primary source for core development on the Mattermost platform; it's written in Go and React, runs as a single Linux binary, and relies on PostgreSQL. A new compiled version is released under an MIT license every month on the 16th.
```

Reviewer 2: I like it so much I'll borrow this for the docs too :)
